apps/examples/kernel_sample/workqueue.c

```diff
@@ -106,5 +107,12 @@ int workqueue_main(int argc, char *argv[])
 
 	printf("test4 is queued and will excute it after 90 ticks\n");
 	work_queue(HPWORK, test_wq4, wq_test4, NULL, 90);
-	return 0;
+	
+	usleep(SECOND_USEC);
```

Comment:
why do you use usleep with huge value?
how about using sleep?

Reply:
sleep(1)? yes!